### ISSUE TITLE
util feature

### ISSUE THREAD

**Issue by larryns** [zaeleus/noodles#248]
Hi,

I'm relatively new to using noodles, so sorry if I'm overlooking something here. But I don't see the "util" feature listed in the noodles crate. Is this intentional?

Thanks a lot for your help,
Larry.

**Comment by zaeleus** [comment]
Yes, this is intentional. noodles-util has no features enabled by default, requiring you to add it as a dependency regardless to define them.

**Comment by larryns** [comment]
Thanks for the fast response. So if I wanted to write a ```RecordBuf``` variable to a SAM record, I would use the ```write_record``` method from ```noodles_util:::alignment::io::writer::builder::Builder```. Is that correct?

Thanks so much.

**Comment by zaeleus** [comment]
If you only want to write out as SAM, use [`noodles::sam::io::Writer`](https://docs.rs/noodles/0.67.0/noodles/sam/io/struct.Writer.html), e.g, https://github.com/zaeleus/noodles/blob/4f7254271f3dad76bae9994a866caa06511aaad7/noodles-sam/examples/sam_write.rs#L48-L49

If you want to dynamically select the output format (SAM, BAM, or CRAM), use [`noodles_util::alignment::io::Writer`](https://docs.rs/noodles-util/0.39.0/noodles_util/alignment/io/writer/struct.Writer.html), e.g., https://github.com/zaeleus/noodles/blob/4f7254271f3dad76bae9994a866caa06511aaad7/noodles-util/examples/util_alignment_rewrite.rs#L36-L39

**Comment by larryns** [comment]
Thank you!
